udata/tests/api/test_datasets_api.py

```diff
@@ -1325,3 +1326,27 @@ def test_resource_types_list(self):
         response = self.get(url_for('api.resource_types'))
         self.assert200(response)
         self.assertEqual(len(response.json), len(RESOURCE_TYPES))
+
+
+@pytest.mark.usefixtures('clean_db')
+class DatasetSchemasAPITest:
+    modules = ['core.dataset']
```

Comment:
I discovered adding this was key to get the routing of `api.get` working, can you explain why?

Reply:
It make the dataset module (and thus it's API) available for the test class. By default we try not to load too much stuff, in order to have faster tests.